engine/classes/Elgg/Cache/MetadataCache.php

```diff
@@ -140,17 +140,11 @@ public function populateFromEntities($guids) {
 		// could be useful at some point in future
 		//$guids = $this->filterMetadataHeavyEntities($guids);
 
-		$db_prefix = _elgg_services()->db->getTablePrefix();
 		$options = array(
 			'guids' => $guids,
 			'limit' => 0,
 			'callback' => false,
 			'distinct' => false,
-			'joins' => array(
-				"JOIN {$db_prefix}metastrings v ON n_table.value_id = v.id",
-				"JOIN {$db_prefix}metastrings n ON n_table.name_id = n.id",
-			),
-			'selects' => array('n.string AS name', 'v.string AS value'),
 			'order_by' => 'n_table.entity_guid, n_table.time_created ASC, n_table.id ASC',
```

Comment:
I know, just wondering if we should drop it in the future unless given explicitly.